ampd/src/monitoring/mod.rs

```diff
@@ -0,0 +1,69 @@
+use std::net::{Ipv4Addr, SocketAddrV4};
+
+use serde::{Deserialize, Serialize};
+
+mod endpoints;
+pub mod server;
+
+#[derive(Clone)]
+pub enum MetricsMsg {
+    IncBlockReceived,
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, Debug)]
+pub struct Config {
+    pub enabled: bool,
+    pub bind_address: SocketAddrV4,
+}
+
+impl Default for Config {
+    fn default() -> Self {
+        Self {
+            enabled: false,
+            bind_address: SocketAddrV4::new(Ipv4Addr::new(127, 0, 0, 1), 3000),
+        }
+    }
+}
+
+impl Config {
+    pub fn get_bind_addr(&self) -> Option<SocketAddrV4> {
```

Comment:
```suggestion
    pub fn bind_addr(&self) -> Option<SocketAddrV4> {
```
Rust convention is not to use `get_`